deployments/Dockerfiles/cluster/run.sh

```diff
@@ -0,0 +1,50 @@
+#! /bin/bash
```

Comment:
Is this identical to the other run.sh? If so, we might want to come up with a smarter approach to avoid duplication